### PR TITLE
[feat]:adding signIn and signUp pages

### PR DESCRIPTION
**Pull request by ultronjs**
Adding Sign-In and Sign-Up page and changed the log-In button to logout button on cart and wishlist page

### CODE REVIEW

File: app.css [hunk 1]
```diff
@@ -31,6 +31,55 @@ body{
...
+.login_password_label{
+    display: flex;
+    justify-content: space-between;
+}
+.input_group{
+    width: 80%;
+}
+.input_icon{
+    position: relative;
+    bottom: 30px;
```
Comment (Kothalkarpriya) [inline app.css]: form more that 10px we have to use rem

File: app.css [hunk 2]
```diff
@@ -31,6 +31,55 @@ body{
...
+/*Form Container for the form such as (Loign,SignUp)*/
+.form_container{
+    padding-top: 4.5rem;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.form{
+    background-color: white;
```
Comment (psychoactiv) [inline app.css:43]: You can use color variable here

**Review by Kothalkarpriya — APPROVED** [review]
I gave a suggestion, you are good to go

**Review by psychoactiv — COMMENTED** [review]
Code is good to go, just do the below change

**Review by psychoactiv — APPROVED** [review]

File: app.css (hunk 2 above)
Comment (ultronjs) [reply app.css:43]: done

File: app.css (hunk 1 above)
Comment (ultronjs) [reply app.css]: done

**Comment by ultronjs** [comment]
Merging after 2 approval and resolving the comments

**Pull request merged**